packages/apps-config/src/endpoints/testing.ts

```diff
@@ -316,10 +316,10 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
       }
     },
     {
-      info: 'oak-testnet',
-      text: t('rpc.test.oak', 'OAK Testnet', { ns: 'apps-config' }),
+      info: 'neumann',
```

Comment:
When we register to polkadot ss58, we register the prefix for Turing Network. 

Neumann as a test can use the same prefix. This is what Astar Network did, using the same prefix as registered ones.

Let’s use the prefix 27 for "Turing Network", as that’s the first available in the front (from 0).